### ISSUE TITLE
'TINY' issue with new master.

### ISSUE THREAD

**Issue by MartynKeigher** [Shopify/dashing#124]
Hey all,

So I used this guide to install the previous dashing master (http://wiki.manageditby.us/dashing-project/), and it worked great!!

Having a slight trouble with the new one when I try to create my own bundle. I basically ran through the same instructions on this link to get a dash going on the new master, but even though all the commands seem to go through OK...I dont get a dashboard that I created. 

I would of course prefer my own DASH to be the default, but right now i cant get ti to show up. I DID call it mklab_dash, i bundled it, but no dice. :(

Basically..a few a kinks in the new one that I could do with some help with.

Thanks :)
